### PR TITLE
APP-2922-sym-parser-emojis-bug Fixed emojis bug

### PR DESCRIPTION
**Pull request by el-youness**
Done by checking the existence of the 'id' parameter in the 'data' node of the payload.

### CODE REVIEW

File: sym_api_client_python/processors/sym_message_parser.py [hunk 1]
```diff
@@ -60,6 +60,14 @@ def get_im_name(self, message_data):
     def get_stream_id(self, message_data):
         return message_data['stream']['streamId']
 
+    def get_tags(self, json_nodes, tag_type):
```
Comment (symphony-hong) [inline sym_api_client_python/processors/sym_message_parser.py]: I would make this method private `def _get_tags(self, json_nodes, tag_type):` like what has been done in Java sdk: [getTags](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/6fd04cd6f09ffa10d8926667cfe350e570bd6bc0/symphony-bdk-legacy/symphony-api-client-java/src/main/java/utils/SymMessageParser.java#L72)

File: sym_api_client_python/processors/sym_message_parser.py [hunk 2]
```diff
@@ -60,6 +60,14 @@ def get_im_name(self, message_data):
...
+        tags = []
+        if json_nodes:
+            for k, v in json_nodes.items():
+                if 'id' in v and len(v['id']) > 0 and v['id'][0]['type'] == self.MENTION_TYPE:
```
Comment (symphony-hong) [inline sym_api_client_python/processors/sym_message_parser.py]: the condition should be: `v['id'][0]['type'] == tag_type`, isn't it?

Comment (symphony-hong) [reply sym_api_client_python/processors/sym_message_parser.py]: do you think we need to check: `'type' in v['id'][0]`? (and `value` as well)

**Review by symphony-hong — CHANGES_REQUESTED** [review]
Some comments added

File: sym_api_client_python/processors/sym_message_parser.py (hunk 1 above)
Comment (el-youness) [reply sym_api_client_python/processors/sym_message_parser.py]: done.

File: sym_api_client_python/processors/sym_message_parser.py (hunk 2 above)
Comment (el-youness) [reply sym_api_client_python/processors/sym_message_parser.py]: Fixed.
For the question. I do not know the purpose of those conditions but I added them anyways to stay consistent with the Java SDK.

**Review by symphony-hong — APPROVED** [review]
LGTM!

**Review by symphony-elias — APPROVED** [review]
LGTM, maps the logic in legacy java SDK

**Pull request merged**